index.md

```diff
@@ -73,7 +75,7 @@ I use this method along with…
 
 #### Asana
 
-I was introduced to [Asana](https://asana.com/) by a [friend](https://twitter.com/neuroamanda) around a year ago, and have been using it on and off for long-term planning of my activities.
+I was introduced to [Asana](https://asana.com/) by a [friend](https://twitter.com/neuroamanda) and have been using it on and off for long-term planning of my activities.
 As a PhD student, you will often need to plan well into the future, with many external deadlines looming ahead: progress reports, vivas, conference submissions etc.
```

Comment:
Re. `.md` and `.doc(x)`, I've added:

> Of course, I don't have the full version history with my `.doc(x)` files, but since I use this format mainly for receiving comments, I save all copies of these files in a separate folder.